### PR TITLE
Corporate Mining Ruin Rework

### PR DESCRIPTION
**Pull request by Erikafox**
## About The Pull Request
Reworks the Corporate Mining ruin into something less. Soulful. The base is now fully atmos piped, comes with a solar installation, decals, real floors (holy shit), and monsters to fight for extra loot. Consider stopping by an energy signal to rescue the poor souls who've been consigned to work here.

You can now find:
A bit of gibonite.
An autolathe board and some parts.
3 mining hardsuits.
5 dead miners of loot. These will require fighting to get to.
A frontier equipment vendor.
A stechkin and 1 spare magazine.
A jukebox
A kinetic accelerator.

- [ ] I affirm that I have tested all of my proposed changes and that any issues found during tested have been addressed.

## Why It's Good For The Game
This ruin is on the list of ruins that needed reworks and I wanted to try my hand at it. I think this is superior to the previous version.

## Changelog

:cl:
add: More areas (for the reworked Corporate Mining ruin)
tweak: Reworks the Corporate Mining Module ruin, both modernizing it and giving it a more defined identity, that of a cramped but workable mining facility.
del: Soul
/:cl:

### CODE REVIEW

**Comment by Erikafox** [comment]
I've been having trouble getting it to spawn for actual testing.

**Comment by ZephyrTFA** [comment]
> I've been having trouble getting it to spawn for actual testing.

use the spawn planet verb and select it as the forced ruin

**Comment by Erikafox** [comment]
I was trying that, my favorite was when it spawned in a jungle world.

File: config/spaceruinblacklist.txt [hunk 1]
```diff
@@ -40,4 +40,4 @@
 #_maps/RandomRuins/SpaceRuins/whiteshipdock.dmm
 #_maps/RandomRuins/SpaceRuins/whiteshipruin_box.dmm
 #_maps/RandomRuins/SpaceRuins/forgottenship.dmm
-
+#_maps/RandomRuins/SpaceRuins/corporate_mining.dmm
```
Comment (Latentish) [inline config/spaceruinblacklist.txt]: While I appreciate this, this is unneccessary at the moment, as we plan to make a ruin database instead.

File: code/game/area/areas/ruins/space.dm [hunk 2]
```diff
@@ -543,3 +543,41 @@
 	name = "Syndicate Laboratory 4071 Surrounding Area"
 	icon_state = "dk_yellow"
 	color = "#26773a88"
+
+//Corporate Mining Ruin
```
Comment (Latentish) [inline code/game/area/areas/ruins/space.dm:553]: You REALLY don't need this many areas. Please cut down or use proper generics for the planet it's on.

**Review by Latentish — DISMISSED** [review]
This map has an extremely high chance of spawning space tiles due to the fact that it has anything on top of turf and area passthroughs. Turf Passthrough must ALWAYS be paired with Area Passthrough, and must have nothing else occupying the tile with it. Otherwise it has a chance to spawn space tiles that don't have proper areas and lighting. This is a problem even on space maps.

Any form of non-turf catwalk must have some kind of turf underneath it. Otherwise it'll likely become overlayed on top of closed turfs (walls), this is bad.

Also what are the 'X's that you've placed, and what are those pit things on the sand?

One last tip, when taking a screenshot from SDMM, use the built in screenshot button so it's a proper resolution, it makes it hard to see what you've put down when you take a regular screengrab.

**Comment by Latentish** [comment]
Please update your image.

**Comment by Latentish** [comment]
Pretty solid. Just cut down on areas.

**Comment by Latentish** [comment]
this ruin is rad. And ready for merging I feel.

**Pull request merged**